great_expectations/cli/datasource.py

```diff
@@ -195,7 +203,7 @@ def _datasource_new_flow(
         selected_files_backend = _prompt_for_execution_engine()
         helper = _get_files_helper(
             selected_files_backend,
-            context_root_dir=context.root_directory,
+            context_root_dir=context.root_directory,  # type: ignore[arg-type] # could be None
```

Comment:
Seems like we could the `root_directory` for `FileDataContext` should never be `None`.
I'll try to correct this in a followup.